### DISCUSSION TITLE
Particle Muffler

### DISCUSSION THREAD

**Discussion by shadowdragon409** [ACGaming/UniversalTweaks#201] [Tweak Suggestions]
I was talking with some people in the Divine Journey 2 Discord and they told me to come over here and pitch this idea to you. 
A whole bunch of mods adds a lot of particle effects that greatly reduce FPS, and there isn't any way to remove them outside of disabling every particle effect in the game. 
I thought it would be cool if there was a block or some universal UI that could disable either all particles or certain particles from spawning either globally, or in a specific radius. Similarly to how the Sound Muffler from "Super Sound Muffler: Revived" works.

**Comment by Primitive-Human** [comment]
More something like [Particle Control](https://www.curseforge.com/minecraft/mc-mods/particle-control)?
